python/ray/air/_internal/usage.py

```diff
@@ -280,3 +280,27 @@ def tag_ray_air_env_vars() -> bool:
         return True
 
     return False
+
+
+def tag_air_entrypoint(
+    trainer_api: bool, tuner_api: bool, run_experiments_api: bool
+) -> None:
+    """Records the entrypoint to an AIR training run.
+
+    Args:
+        trainer_api: Whether or not the user went through the Trainer.fit API
+        tuner_api: Whether or not the user went through the Tuner.fit API
+            NOTE: Because Trainer.fit goes through Tuner.fit, both of the flags above
+            could be True, in which case we know the user is running a single Train run.
+        run_experiments_api: Whether or not the user went through `tune.run_experiments`
+    """
+    if run_experiments_api:
+        tag = "tune.run_experiments"
+    elif not trainer_api and not tuner_api:
+        tag = "tune.run"
+    elif tuner_api and not trainer_api:
+        tag = "Tuner.fit"
+    elif trainer_api:
+        tag = "Trainer.fit"
```

Comment:
what do you think if we don't add this util, and just record the raw flag values.
it's usually easier or more flexible to work with the raw data in data analysis.

Reply:
Oh, you mean save the True/False flags? I think Kai's suggestion about consolidating all of these actually makes sense -- there are 3 sets of flags now. Would rather just save a single thing.